src/Resources/translations/ResetPasswordBundle.pl.xlf

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+    <file original="file.ext" source-language="en" datatype="plaintext">
+        <body>
+            <trans-unit id="1">
+                <source>%count% year|%count% years</source>
+                <target>%count% roku|%count% lat</target>
```

Comment:
This is broken. The `pl` local has 3 pluralization forms, not 2.

Reply:
But here, in this context, it is used in genitive, so we need only to forms here. If not please provide me with the sentence, context in which these will be used.

Reply:
@thomas2411 checkout https://github.com/SymfonyCasts/verify-email-bundle/pull/56

Reply:
@stof, @jrushlow I have commented it in https://github.com/SymfonyCasts/verify-email-bundle/pull/56#discussion_r577870396

Reply:
thanks @thomas2411 - what @stof is saying (correct me if im wrong) its not so much if it should be `roku` | `lat` its the symfony uses 3 pluralization params for `PL`. so in this context we must provide 3.. which could be
```xml
<source>%count% year|%count% years</source>
<target>%count% roku|%count% lat|%count% lat</target>
```

Reply:
OK, so just copy paste the second one into third one. So second and third would be the same in this context.

Reply:
I will make the change.